pages/index.tsx

```diff
@@ -1,9 +1,9 @@
-import Status from '@/components/StatusFilter';
-import NFTView from '@/view/NFTView';
-import Head from 'next/head';
 import { useState } from 'react';
+import Head from 'next/head';
+
+import NFTView from '@/view/NFTView';
 
-const TABINFORM = [
+const TAB_INFORMS = [
```

Comment:
IN과 FORMS에도 언더바로 끊어주기!

```suggestion
const TAB_IN_FORMS = [
```